packages/blade/src/components/Typography/Display/__tests__/Display.web.test.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
```

Comment:
add ssr test

Reply:
We need to add this for other typography components too, I can add them in a separate PR.